src/providers/elevenlabs_agent.py

```
@@ -386,6 +389,29 @@ async def send_interrupt(self) -> None:
+                "Do not add, remove, or paraphrase words. "
+                f"<message>{text}</message>"
+            ),
+        }
+        try:
+            await self._ws.send(json.dumps(message))
+            logger.info(
+                f"[elevenlabs] [{self._call_id}] Sent no-input announcement request",
+                text_preview=text[:80],
+            )
+            return True
+        except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)